pkg/apis/scope/v0alpha1/types.go

```diff
@@ -21,11 +21,23 @@ type ScopeSpec struct {
 }
 
 type ScopeFilter struct {
-	Key      string `json:"key"`
-	Value    string `json:"value"`
-	Operator string `json:"operator"`
+	Key      string         `json:"key"`
+	Value    string         `json:"value"`
+	Operator FilterOperator `json:"operator"`
 }
 
+// Type of the filter operator.
+// +enum
+type FilterOperator string
+
+// Defines values for FilterOperator.
+const (
+	FilterOperatorEquals        FilterOperator = "equals"
```

Comment:
Is it ok for this to be `equals`, `not-equals` etc.? Or should they be `==`, `!=` etc.

Reply:
I prefer text because `=` assigns values in go/typescript bug means equal in promql. I'm sure other databases have their own opinions.